Source/OIDRegistrationRequest.m

```diff
@@ -141,8 +143,8 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
   NSString *tokenEndpointAuthenticationMethod =
       [aDecoder decodeObjectOfClass:[NSString class]
                              forKey:OIDTokenEndpointAuthenticationMethodParam];
-  NSSet *additionalParameterCodingClasses = [NSSet setWithArray:@[ [NSDictionary class],
-                                                                   [NSString class] ]];
+  additionalParameterCodingClasses = [NSSet setWithArray:@[ [NSDictionary class],
```

Comment:
Can we introduce a new variable name here rather than re-using the previous one, I feel like it makes it simpler to read that way.

Reply:
Yes sure, I'll do that!